frontend/src/App.tsx

```diff
@@ -4,7 +4,7 @@ import logoutIcon from '@iconify/icons-mdi/logout';
 import darkIcon from '@iconify/icons-mdi/weather-night';
 import lightIcon from '@iconify/icons-mdi/weather-sunny';
 import { Icon } from '@iconify/react';
-import { Box, Button } from '@material-ui/core';
+import { Box } from '@material-ui/core';
```

Comment:
Nitpick: Typo on `separate` in the title.

Reply:
fixed